src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, job_id, initial_params, area):
         self.device_statistics = DeviceStatistics()
         self.file_export_endpoints = FileExportEndpoints()
         self.kpi = KPI()
+        self.baseline_peak_stats = BaselinePeakEnergyStats()
```

Comment:
Would be better if renamed  `baseline_peak_stats` to `area_throughput_stats`

Reply:
ok! Will do!